File: services/23-symfony/src/Controller/HealthController.php
<?php

namespace App\Controller;

use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
use Symfony\Component\HttpFoundation\JsonResponse;
use Symfony\Component\Routing\Annotation\Route;

class HealthController extends AbstractController
{
    #[Route('/', name: 'hello', methods: ['GET'])]
    public function hello(): JsonResponse
    {
        return $this->json(['message' => 'Hello from Symfony 7.2', 'framework' => '23-symfony', 'version' => '1.0.0']);
    }

    #[Route('/health', name: 'health', methods: ['GET'])]
    public function health(): JsonResponse
    {
        return $this->json(['status' => 'ok', 'version' => '1.0.0']);
    }

    #[Route('/health/live', name: 'liveness', methods: ['GET'])]
    public function liveness(): JsonResponse
    {
        return $this->json(['status' => 'ok']);
    }

    #[Route('/health/ready', name: 'readiness', methods: ['GET'])]
    public function readiness(): JsonResponse
    {
        return $this->json(['status' => 'ok']);
    }
}

File: services/21-phoenix/lib/app_web/controllers/health_controller.ex
defmodule AppWeb.HealthController do
  use Phoenix.Controller, formats: [:json]

  def hello(conn, _params) do
    json(conn, %{message: "Hello from Phoenix 1.7", framework: "21-phoenix", version: "1.0.0"})
  end

  def health(conn, _params), do: json(conn, %{status: "ok", version: "1.0.0"})
  def liveness(conn, _params), do: json(conn, %{status: "ok"})
  def readiness(conn, _params), do: json(conn, %{status: "ok"})
end
